contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -232,7 +245,8 @@ contract ProtocolShareReserve is AccessControlledV8, ReentrancyGuardUpgradeable,
         address destination,
         address asset
     ) external view returns (uint256) {
-        for (uint256 i = 0; i < distributionTargets.length;) {
+        _ensureMaxLoops(distributionTargets.length);
```

Comment:
I think we don't need this check if we do the check I commented for `addOrUpdateDistributionConfigs`